tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py{27,34,35}-django19
+skipsdist = True
+
+[tox:travis]
+2.7 = py27
+3.4 = py34
+3.5 = py35
+
+[testenv]
+passenv = *
+deps = -rrequirements.txt
+
+commands =
+    python manage.py check
+    # pytest --flake
+    pytest --cov-report= --cov=auth_helpers,bulk_adding,cached_counts,candidates,compat,elections,moderation_queue,official_documents,results,tasks,uk_results
```

Comment:
Can we pass `--doctest-modules` here to include the doctests (although as we discussed yesterday, porting them to unittests is probably the longer term goal)

Reply:
Also maybe for a large project like this it would be nicer to define the coverage targets in a `.coveragerc`

Reply:
So adding `--doctest-modules` hits what looks like this bug for me on python 2.7: https://github.com/pytest-dev/pytest/issues/2854 and that in turn looks like it's this bug in python: https://bugs.python.org/issue25532.

I reckon it's not worth adding for now…

Reply:
:+1: ignore and we'll cover it under #328